azure-pipelines.yml

```diff
@@ -161,52 +161,20 @@ stages:
                 /p:TeamName=DotNetCore
                 /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
               displayName: Sign packages
-        - job: Run_SDL
-          displayName: Run SDL tool
-          variables:
-            - group: SDL_Eng_KeyVault
-            - _BuildConfig: Release
-          steps:
-            - checkout: self
-              clean: true
-            - powershell: eng\common\build.ps1
-                -configuration $(_BuildConfig)
-                -restore
-                -prepareMachine
-                -ci
-                /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
-              displayName: Windows Build / Publish
-            - task: NuGetToolInstaller@1
-              displayName: 'Install NuGet.exe'
-            - task: NuGetCommand@2
-              displayName: 'Install Guardian'
-              inputs:
-                restoreSolution: $(Build.SourcesDirectory)\eng\common\sdl\packages.config
-                feedsToUse: config
-                nugetConfigPath: $(Build.SourcesDirectory)\eng\common\sdl\NuGet.config
-                externalFeedCredentials: GuardianConnect
-                restoreDirectory: $(Build.SourcesDirectory)\.packages
-            # Script to execute SDL tools.
-            # TsaNotificationEmail will be changed to dnceng@
-            # to check what the noise level would be with regular SDL runs.
-            - powershell: eng/common/sdl/execute-all-sdl-tools.ps1
-                -GuardianPackageName Microsoft.Guardian.Cli.0.3.2
-                -NugetPackageDirectory $(Build.SourcesDirectory)\.packages
-                -Repository $(Build.Repository.Name)
-                -SourceDirectory $(Build.SourcesDirectory)
-                -ArtifactsDirectory $(Build.SourcesDirectory)
-                -DncEngAccessToken $(dn-bot-dotnet-build-rw-code-rw)
-                -SourceToolsList @("policheck","credscan")
-                -TsaInstanceURL "https://devdiv.visualstudio.com/"
-                -TsaProjectName "DEVDIV"
-                -TsaNotificationEmail "subal@microsoft.com"
-                -TsaCodebaseAdmin "REDMOND\subal"
-                -TsaBugAreaPath "DevDiv\NET\NET Core "
-                -TsaIterationPath "DevDiv"
-                -TsaRepositoryName "Arcade"
-                -TsaCodebaseName "Arcade"
-                -TsaPublish $True
-              displayName: 'Execute SDL'
+  - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+    - template: /eng/common/templates/job/execute-sdl.yml
+      parameters:
+        additionalParameters: ' -ArtifactsDirectory $(Build.SourcesDirectory)
+          -SourceToolsList @("policheck","credscan")
+          -TsaInstanceURL "https://devdiv.visualstudio.com/"
```

Comment:
Is devdiv correct compared to dnceng?

Reply:
Yes, because this is the only account that is onboarded to TSA and .NET core uses this account for their SDL bugs. so we are doing the same.